src/sinks/elasticsearch.rs

```diff
@@ -834,6 +835,24 @@ mod integration_tests {
         .await;
     }
 
+    #[test]
```

Comment:
I think you'll want to use `tokio::test` here since it is an async function.

Reply:
I don't think it needs the async, so I'll drop it down to a normal function or do you prefer all tests be `tokio::test`?

Reply:
Ah, you are right, this one doesn't need async. I notice this is just parsing the configuration and making an assertion on it so it could be moved up to the `mod test` here. `mod integration_test` is generally reserved for tests that need external resources (usually a docker container running) to run. Here these tests depend on a running ES instance, which this test doesn't need.